docs/en/modules/rfswitch.md

```diff
@@ -0,0 +1,52 @@
+# rfswitch Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2016-10-27 | [Roman Fedorov](https://github.com/ffedoroff) | [Roman Fedorov](https://github.com/ffedoroff) | [rfswitch.c](../../../app/modules/rfswitch.c)|
+
+
+Module for operate 433/315Mhz devices like power outlet sockets, relays, etc.
+This will most likely work with all popular low cost power outlet sockets
+with a SC5262 / SC5272, HX2262 / HX2272, PT2262 / PT2272, EV1527,
+RT1527, FP1527 or HS1527 chipset.
+
+This module using some code from original [rc-switch](https://github.com/sui77/rc-switch/) arduino lib
+but NodeMCU and Arduino are not fully compatible, and it cause
+for full rewrite **rc-switch** into this new **rfswitch** lib for NodeMCU with Lua support.
+
+Connection of transmitter:
+
+| Transmitter  | ESP8266  | comments                        |
+| :----------- | :------- | :------------------------------ |
+| vin or + | 3V3 | 3.3 - 5 volts on ESP8266 or other power source |
+| ground or - | GND | ground should be connected to ESP8266 and to power source |
+| data pin | 6 | almost any pin on ESP8266 |
+
+You can read more about connection, [here](https://alexbloggt.com/wp-content/uploads/2015/10/nodemcu_433_transmitter.png) or [here](https://alexbloggt.com/funksteckdosensteuerung-mit-esp8266/).
+
```

Comment:
Can you make a note somewhere that this is synchronous and hence the total delay should be kept to a reasonable value. For examples, setting the repeat count to a large value will cause problems. The recommended limit is maybe 20ms. It might be better (for a user) to call this function once and then call it asynchronously a few more times (e.g. using node.task.post)